core/templates/services/guppy-initialization.service.ts

```diff
@@ -35,16 +37,30 @@ export class GuppyInitializationService {
   private guppyInstances: Array<GuppyObject> = [];
   private onScreenKeyboardShown = false;
 
-  init(guppyDivClassName: string): void {
+  init(guppyDivClassName: string, initialValue = ''): void {
```

Comment:
Missing typeinfo -- shouldn't a ts checker catch this? /cc @nishantwrp @vojtechjelinek

Reply:
Again the type is inferred here due to `initialValue = ''` i.e. string.

Reply:
Ah ok, thanks. What do you want the convention to be in the codebase going forward, though? Should all types be specified, ideally?

Reply:
Hmm, I'm fine with both the possibilities here. But in cases where there is no default value like here, there should be specific types.

Reply:
Ah ok, thanks. Please mention the full desired convention on your types wiki page then, since it will be used as a reference in the future for any type-related issues.  Thanks!

Reply:
Done